ddp-workspace/projects/ddp-dsm-ui/src/app/participant-list/participant-list.component.ts

```diff
@@ -706,6 +708,36 @@ export class ParticipantListComponent implements OnInit {
     });
   }
 
+
+  private deleteFiltersAccordingToPermission(): void {
+    let columnNamesToDelete: string[];
+
+    if(this.mrAndDssFalse) {
+      columnNamesToDelete = ['k', 'address', 'm', 'oD'];
+    } else if(this.mrFalseDssTrue) {
+      columnNamesToDelete = ['m', 'oD'];
```

Comment:
shouldn't tissue be here as well?

Reply:
we checked that and looks like the tissue is already hidden somewhere on the top